.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+env: # These should match the tox env list
+    - TOXENV=py27
+install: pip install coveralls tox
+script: tox
+# Special snowflake.  Our tests depend on a credentials_secret.json
+before_install:
+    - cp tests/json/credentials.json tests/json/credentials_secret.json
```

Comment:
Hm, I'm not convinced this will be enough; I wouldn't expect `tests/json/credentials.json` to have legit creds.  Poking.